### DISCUSSION TITLE
Is it possible to have each TabViewItem content(UI) running in its own process?

### DISCUSSION THREAD

**Discussion by harvinders** [microsoft/WindowsAppSDK#1642] [Q&A]
We are creating an application which hosts plugins with their views in a TabView. I would like to understand 

1. Is it possible, today, to host those views in their own processes, similar to what the edge does with tabs? 
2. If not today, are there something in the roadmap that would enable it? What should we be looking out for on the roadmap? AppWindow?
3. If it is possible or would be possible in future, would that mean, we can have those views based on different versions of WinAppSDK?

**Comment by btueffers** [comment]
Hi @harvinders, thank you for starting this discussion! This topic is more appropriate for the WinUI repo. Please open an [issue](https://github.com/microsoft/microsoft-ui-xaml/issues) with them to get the most beneficial response.
